statsmodels/tsa/statespace/tests/test_sarimax.py

```diff
@@ -1642,87 +1619,87 @@ def test_manual_stationary_initialization():
 
     # Create the first model to compare against
     mod1 = sarimax.SARIMAX(endog, order=(3,0,0))
-    mod1.update([0.5,0.2,0.1,1])
-    res1 = mod1.filter()
+    res1 = mod1.filter([0.5,0.2,0.1,1])
 
     # Create a second model with "known" initialization
     mod2 = sarimax.SARIMAX(endog, order=(3,0,0))
-    mod2.update([0.5,0.2,0.1,1])
-    mod2.initialize_known(res1.initial_state, res1.initial_state_cov)
+    mod2.ssm.initialize_known(res1.ssm_result.initial_state,
+                              res1.ssm_result.initial_state_cov)
     mod2.initialize_state()  # a noop in this case (include for coverage)
-    res2 = mod2.filter()
+    res2 = mod2.filter([0.5,0.2,0.1,1])
 
     # Create a third model with "known" initialization, but specified in kwargs
     mod3 = sarimax.SARIMAX(endog, order=(3,0,0),
                            initialization='known',
-                           initial_state=res1.initial_state,
-                           initial_state_cov=res1.initial_state_cov)
-    mod3.update([0.5,0.2,0.1,1])
-    res3 = mod3.filter()
+                           initial_state=res1.ssm_result.initial_state,
+                           initial_state_cov=res1.ssm_result.initial_state_cov)
+    res3 = mod3.filter([0.5,0.2,0.1,1])
 
     # Create the forth model with stationary initialization specified in kwargs
     mod4 = sarimax.SARIMAX(endog, order=(3,0,0), initialization='stationary')
-    mod4.update([0.5,0.2,0.1,1])
-    res4 = mod4.filter()
+    res4 = mod4.filter([0.5,0.2,0.1,1])
 
     # Just test a couple of things to make sure the results are the same
     assert_almost_equal(res1.llf, res2.llf)
-    assert_almost_equal(res1.filtered_state, res2.filtered_state)
+    assert_almost_equal(res1.ssm_result.filtered_state,
```

Comment:
general question, not here
Aren't we using `results` (plural) as a name, i.e. `ssm_results` instead of `ssm_result`?
(should be consistent to avoid typos, although we usually have autocompletion)